lib/rails-jquery-autocomplete/version.rb

```diff
@@ -1,3 +1,3 @@
 module RailsJQueryAutocomplete
-  VERSION = '1.0.0'
+  VERSION = '1.1.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.